### PR TITLE
use https links

### PR DESCRIPTION
**Pull request by bmwiedemann**
whereever possible
and fix houndci URL

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#   http://www.apache.org/licenses/LICENSE-2.0
+#   https://www.apache.org/licenses/LICENSE-2.0
```
Comment (vuntz) [inline Gemfile]: Please no: look at the content at the URL and see the end, the recommendation is to use the http URL. Changing this to https might break automatic license detection.

**Comment by bmwiedemann** [comment]
updated

File: scripts/barclamp_mgmt_lib.rb [hunk 2]
```diff
@@ -229,7 +229,7 @@ def generate_navigation
     out.puts '# you may not use this file except in compliance with the License.'
     out.puts '# You may obtain a copy of the License at'
     out.puts '#'
-    out.puts '#   http://www.apache.org/licenses/LICENSE-2.0'
+    out.puts '#   https://www.apache.org/licenses/LICENSE-2.0'
```
Comment (vuntz) [inline scripts/barclamp_mgmt_lib.rb]: You forgot that one :-)

**Comment by vuntz** [comment]
+1 after fixing the last comment.

**Comment by rsalevsky** [comment]
+1

**Pull request merged**